frontend/src/pages/dashboard/ManualTradingPage.tsx

```
@@ -308,6 +367,144 @@ const ManualTradingPage: React.FC = () => {
+        assetTiers: normalizeArray(filters.assetTiers),
+        strategyIds: normalizeArray(filters.strategyIds),
+      };
+    },
+    []
+  );
+
+  const handleStartOpportunityScan = useCallback(() => {
+    const normalized = normalizeFiltersForSubmission(scanFilters);
+    setScanDialogOpen(false);
+    void handleConsensusAction('opportunity', { filters: normalized });
+  }, [handleConsensusAction, normalizeFiltersForSubmission, scanFilters]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)